src/ReportManager/SchemaFormContext/utils.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
I just realized I was doing a circular reference with this utils methods, I'm refactoring the code to avoid this, also realized I was missing logic of active/inactive fields for rendering and form validation, on it 🫡